### PR TITLE
Use my first Project to review

### PR DESCRIPTION
**Pull request by mgmediaweb**
# Project Review

- Install the Linters.
- Install all files from my first project.
- Check all code and repair all errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+      
+      <footer>
+          <a href="#" class="link-dark text-decoration-none mx-2">Privacy</a>
+          <a href="#" class="link-dark text-decoration-none mx-2">Terms</a>
+          <a href="#" class="link-dark text-decoration-none mx-2">Contact</a>
+          <p class="m-0">Copyright 2016, Original Trombones</p>
+      </footer>
+  </div>
+
+    <script  src="js/index.js"></script>
```
Comment (mgmediaweb) [inline index.html]: this file is not being used, I recommend you delete it

File: index.html [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+              <li><a class="nav-link text-decoration-none link-dark" href="#pricing">Pricing</a></li>
+          </ul>
+      </nav>
+  </header>
+  <div class="container">
+      <div class="row my-5">
+          <div class="col text-center my-3">
+              <form id="form" name="form" method="get" action="https://www.freecodecamp.com/email-submit">
+                  <h4 class="fw-bold">Handcrafted, home-made masterpieces</h4>
+                  <input id="email" name="email" type="email" placeholder="Enter you email address" class="form-control mx-auto mt-3" style="width:250px" required />
```
Comment (mgmediaweb) [inline index.html]: Do not recommend using inline style, because, not an efficient way to customize an element, please only use external CSS

**Review by mgmediaweb — COMMENTED** [review]
# Changes Request ♻️ 

- Please read the following comment, and wait for your changes

File: index.html (hunk 2 above)
Comment (Lucas-Masaba) [reply index.html]: - [x] I think it would be better to give a reason why the person should use only external CSS👍.

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @mgmediaweb  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Successfully identified issues with code base ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._


[OPTIONAL]
- I believe adding emojis is an excellent way to enhance communication. An emoji is literally 20 words in one and it also brings comfort to those reading the changes you suggested 👍

File: index.html (hunk 2 above)
Comment (mgmediaweb) [reply index.html]: @Lucas-Masaba I already changed the comment, I must re-send the pull request in the dashboard?

Comment (Lucas-Masaba) [reply index.html]: Yes, please. Send the pull request to the dashboard for a re-review🏂

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @mgmediaweb 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 
:heavy_check_mark: Good commit messages.

- [ ] [OPTIONAL] next time make sure you have at least two comments review

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**